build/charts/antrea/conf/antrea-agent-secondary-network.conf

```diff
@@ -0,0 +1,37 @@
+# Name of the primary OpenVSwitch bridge antrea-agent will create for Secondary interface needs.
+# Make sure it doesn't conflict with your existing OpenVSwitch bridges.
+# Defaults to br-int-secnet1.
+OVSIntegrationBridge: {{ (.Values.secondaryNetworkOVS).ovsIntBrName | quote }}
+
+# Name of the OpenVSwitch bridge antrea-agent will create for Secondary network specific tunnel needs.
+# Make sure it doesn't conflict with your existing OpenVSwitch bridges.
+# Defaults to br-inti2.
+OVSTransportBridge: {{ (.Values.secondaryNetworkOVS).ovsTransBrName | quote }}
+
+# Datapath type to use for the OpenVSwitch bridge created by Antrea. Supported values are:
+# - system
+# - netdev
+# 'system' is the default value and corresponds to the kernel datapath. Use 'netdev' to run
+# OVS in userspace mode. Userspace mode requires the tun device driver to be available.
+OVSDatapathType: {{ (.Values.secondaryNetworkOVS).ovsDatapathType | quote }}
+
+# Name of the OVS patch port which would connected two OVS bridges per secondary inteface config.
+OVSPatchPort: {{ (.Values.secondaryNetworkOVS).ovsPatchPort | quote }}
+
+# Determines how traffic is encapsulated. It has the following options:
+#  encap(default):    Inter-node Pod traffic is always encapsulated and Pod to external network
```

Comment:
@arunvelayutham : do we plan to support these modes for secondary networks? If not now, maybe remove this option or unsupported modes for now.

Reply:
We plan to support encap mode for now @jianjuns . We will clean-up the other mode info for now.

Reply:
Idea is to support all the modes in par with primary network. But, for now I will remove it and get it done as a next step @jianjuns